app/frontend/wallet/shelley/shelley-transaction.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable camelcase */
+import {encode} from 'borc'
+import {blake2b} from 'cardano-crypto.js'
+import bech32 from './helpers/bech32'
+
+const TxOutputTypeCodes = {
+  SIGN_TX_OUTPUT_TYPE_ADDRESS: 1,
+  SIGN_TX_OUTPUT_TYPE_PATH: 2,
+}
+
+function ShelleyTxAux(inputs, outputs, fee, ttl, certs = []) {
+  function getId() {
+    return blake2b(encode(ShelleyTxAux(inputs, outputs, certs, fee, ttl)), 32).toString('hex')
```

Comment:
Using `borc.encode` instead of just `encode` (everywhere) would greatly help me in searching for the places where we use the borc library.